tests/resources/test_resources.py

```diff
@@ -52,7 +52,7 @@ def _validate_access(response, original):
 
         until = arrow.get(embargo["until"]).datetime
         orig_until = arrow.get(orig_embargo["until"]).datetime
-        assert until.strftime("%Y-%m-%d") == orig_until.strftime("%Y-%m-%d")
+        assert until.isoformat() == orig_until.isoformat()
```

Comment:
this got sucked into the updating spree, but the slightly more precise check shouldn't hurt either